### DISCUSSION TITLE
English language support in MySQL console

### DISCUSSION THREAD

**Discussion by VladimirChmakov** [apache/doris#29687] [A - General / Q&A]
Hi! 
I'm trying to use help system from the MySQL console, but the responses are coming out in Chinese. 
Questions:
1. Is English supported? 
2. If it is - how to switch the help system to English?

**Comment by LemonLiTree** [comment]
Where exactly are you referring to?

**Comment by VladimirChmakov** [comment]
1. Open MySQL console, e.g. mysql.exe in MS Windows
2. Type HELP CREATE TABLE;
3. See the text in Chinese

Question: can this be switched to English?

**Reply by LemonLiTree** [reply]
I will fix this problem

**Reply by VladimirChmakov** [reply]
Thank you!
